chromadb/config.py

```diff
@@ -116,6 +116,7 @@ class Settings(BaseSettings):  # type: ignore
     is_persistent: bool = False
     persist_directory: str = "./chroma"
 
+    chroma_memory_limit_bytes: int = 0
```

Comment:
How do I turn this capability on and off? Is 0 implicitly off?

Reply:
yes. 0 is unlimited